P0009/P0009r12.md

```diff
@@ -742,8 +785,8 @@ shall determine._
 >  _Make the following changes to 22.7.1 <b>[views.general]</b>_,
 
 [1]{.pnum} The header `<span>` defines the view span. 
-[The header `<mdspan>` defines the view `basic_mdspan`, the type alias 
-`mdspan`, and other facilities for interacting with these views.]{.add}
+[The header `<mdspan>` defines the view `mdspan`
```

Comment:
should it be saying "the class template `mdspan` and other facility for interacting with these non-owning multidimensional views."

Reply:
agree with damien

Reply:
I implemented this fix -- thanks Damien and Christian!